dap-etl/extraction/src/main/scala/org/broadinstitute/monster/dap/common/OkWrapper.scala

```diff
@@ -44,11 +47,12 @@ class OkWrapper extends HttpWrapper {
         val request = chain.request()
         var response = chain.proceed(request)
         var tryCount = 0
-        while (!response.isSuccessful && tryCount < MAX_RETRIES) {
+        while (!response.isSuccessful && tryCount < OkWrapper.MAX_RETRIES) {
           logger.warn(
             s"Unsuccessful response, retrying [url=${request.url()}, code=${response.code()}, try=${tryCount}]"
           )
           tryCount += 1
+          response.close()
```

Comment:
This is the crux of the issue, we need to close before proceeding on with the chain. I can confirm that the behavior repros in the supplied test when this line is removed.